src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,33 +2,40 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
 
-    public int total = -1;
+    private static final int EMPTY_STACK_HEAD_INDEX = -1;
+    private static final int CAPACITY = 12;
+    private static final int FULL = 11;
```

Comment:
This constant is related to CAPACITY . Changing only one leads to an error (inconsistency).